src/openmc_nek_driver.cpp

```diff
@@ -158,7 +115,7 @@ void OpenmcNekDriver::init_mappings()
     }
 
     // Set element -> cell instance mapping on each Nek rank
-    intranode_comm_.Bcast(elem_to_cell.data(), n_global_elem_, MPI_INT32_T);
+    intranode_comm_.Bcast(elem_to_cell.data(), elem_to_cell.size(), MPI_INT32_T);
     elem_to_cell_ = elem_to_cell;
```

Comment:
Yes, I think you're right. If you don't mind, I'd like to defer on this one because I have more changes already lined up that refactor this method further. I'll make sure to get rid of the `elem_to_cell` local in the next update. Is that ok?

Reply:
Yup, absolutely